### DISCUSSION TITLE
foundry up

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/foundry-full-course-cu#1017] [Q&A] [answered]
i have installed the foundry successfully using this "curl -L https://foundry.paradigm.xyz | bash" in my wsl ubuntu
but when i try to run foundryup it gives me this error
![Uploading image.png…]()
```
foundryup: command not found
```

**Comment by theirrationalone** [comment, marked as answer]
You Might omitted to execute this command...

Note: `username` is your machine's username, please put that on there instead.

```bash
source /Users/username/.bashrc
```
